ui/insight/src/chart.ts

```diff
@@ -98,79 +115,79 @@ function makeChart(el, data) {
           );
         })(),
         shared: true,
-      },
+      } as Highcharts.SeriesTooltipOptions,
```

Comment:
Highcharts' type defintions confuse me: `Highcharts.TooltipOptions` extends `Highcharts.SeriesTooltipOptions`. I'd think it would be the other way around. The way it is now, shared: true gets rejected by typescript and requires a cast.